contrib/test/crio-integration-results.yml

```diff
@@ -0,0 +1,66 @@
+---
+# vim-syntax: ansible
+
+- hosts: '{{ hosts | default("all") }}'
+  vars:
+    crio_integration_filepath: "/root/src/github.com/kubernetes-incubator/cri-o/testout.txt"
+    crio_node_e2e_filepath: "/root/src/k8s.io/kubernetes/node-e2e.log"
+    # Treat empty env. var as "undefined and use default()
+    result_dest_basedir: '{{ lookup("env","RESULTSDIR") |
+                             default(playbook_dir ~ "/results", True) }}'
+    result_filepaths: []  # do not use
+    _dstfnbuff: []  # do not use
+  tasks:
+    - name: The crio_integration_filepath is required
+      tags:
+          - integration
+      set_fact:
+        result_filepaths: "{{ result_filepaths + [crio_integration_filepath] }}"
+
+    - name: The crio_node_e2e_filepath is required
+      tags:
+          - e2e
+      set_fact:
+        result_filepaths: "{{ result_filepaths + [crio_node_e2e_filepath] }}"
+
+    - name: Verify expectations
+      assert:
+        that:
+          - 'result_dest_basedir | default(False, True)'
+          - 'result_filepaths | default(False, True)'
+          - '_dstfnbuff == []'
+          - 'results_fetched is undefined'
+
+    - name: Results directory exists
+      file:
+          path: "{{ result_dest_basedir }}"
+          state: directory
+      delegate_to: localhost
+
+    - name: destination file paths are buffered for overwrite-checking and jUnit conversion
+      set_fact:
+          _dstfnbuff: >
+              {{ _dstfnbuff |
+                 union( [result_dest_basedir ~ "/" ~ inventory_hostname ~ "/" ~ item | basename] ) }}
+      with_items: '{{ result_filepaths }}'
+
+    - name: Overwriting existing results assumed very very bad
+      fail:
+          msg: "Cowardly refusing to overwrite {{ item }}"
+      when: item | exists
+      delegate_to: localhost
+      with_items: '{{ _dstfnbuff }}'
```

Comment:
This will be fatal, no junit output will be produced, CI must be sensitive to exit-status of this playbook.